sql/atxs/atxs.go

```diff
@@ -539,3 +539,30 @@ func CountAtxsByOps(db sql.Executor, operations builder.Operations) (count uint3
 	)
 	return
 }
+
+// IterateForGrading selects every atx from publish epoch and joins identities to load malfeasence proofs if they exist.
+func IterateForGrading(
+	db sql.Executor,
+	epoch types.EpochID,
+	fn func(id types.ATXID, atxtime, prooftime int64, weight uint64) bool,
+) error {
+	if _, err := db.Exec(`
+		select atxs.id as id, atxs.received as atxtime,
+			   identities.received as prooftime, effective_num_units, tick_count from atxs
```

Comment:
Does `as foo` have any effect here? The values are referred to by col index anyway.
```suggestion
		select atxs.id, atxs.received, identities.received, effective_num_units, tick_count
		from atxs
```

Reply:
it doesn't, i added it for my clarify, can drop